scipy/spatial/transform/rotation.py

```diff
@@ -84,3 +85,41 @@ def as_quaternion(self):
             return self._quat[0]
         else:
             return self._quat
+
+    def as_dcm(self):
+        """Return the direction cosine matrix representation of the Rotation.
+
+        This function returns a numpy.ndarray of shape (3 x 3) or (N x 3 x 3)
```

Comment:
Just something what I thought: maybe write shape as `(3, 3)` and `(N, 3, 3)` just to keep it uniform everywhere in the docstrings. Although this notation also looks nice, I don't mind to keep it here. Judge yourself.